docs/Using-Fleet/configuration-files/README.md

```diff
@@ -447,7 +419,7 @@ For more information about integrations and Fleet automations in general, see th
 
 It's recommended to use the Fleet UI to configure integrations since secret credentials (in the form of an API token) must be provided. See the [Automations documentation](../../Using-Fleet/Automations.md) for the UI configuration steps.
 
-#### Organization information
+#### Org info
```

Comment:
I appreciate the consistency between this and the subsequent heading, but I think "Organization information" could be easier for readers to understand — at least when we're introducing a new section. Most native English speakers will get it, but I don't know what translations of "Org info" would look like.

Reply:
@chris-mcgillicuddy prioritizing understandability over consistency makes sense to me.

I updated my PR to use "Organization information"